web/styles/banners.css

```diff
@@ -40,6 +40,11 @@
     grid-area: banner-action-buttons;
     display: flex;
     gap: 0.5em; /* 8px at 16px/1em */
+    /* This margin is required to align the banner action
+       buttons vertically, since we cannot use flexbox
+       center alignment as we also need to account for the
+       UI when the banner has a muli-line label. */
+    margin: 0.0625em 0; /* 1px at 16px/1em */
```

Comment:
I haven't tested, but the sub-pixel margin here for font sizes below 16px makes me wonder if there won't be alignment problems at smaller font sizes. I guess if its just to align the action buttons with themselves, it'll be okay.

Reply:
This is to align the banner action buttons with the banner label, since the height of both of these elements aren't the same; And we can't use center alignment because we also want the buttons to stay at the flex-start in case of multi-line banner label.

Reply:
Cool. Sounds good.